main.go

```diff
@@ -739,6 +738,314 @@ func (u *UserSessionsApp) DeleteRequest(writer http.ResponseWriter, r *http.Requ
 
 // -------- END SESSIONS DATA --------
 
+// -------- START SEARCHES --------
```

Comment:
This isn't necessary for this pull request, but we should consider splitting this stuff out into multiple separate files, all of which are part of the main package.

Reply:
Noted in CORE-9198 and assigned to myself.